README.md

```diff
@@ -70,3 +98,61 @@ For more information, see (some of these pages are in Dutch):
 - [Amsterdam Schema Validator 👩🏼‍🏫](https://observablehq.com/@bertspaan/amsterdam-schema-validator)
 - [Werkbestand Team Dataservices](https://observablehq.com/@bertspaan/werkbestand-team-dataservices)
 - [Amsterdam Schema Playground 🎠](https://observablehq.com/@bertspaan/amsterdam-schema-playground)
+
+# Publishing
+
+In order to publish the Amsterdam Schema to the object store
+install the Python package included in this repository:
+
+```console
+% python3.8 -m venv venv
+% pip install -U pip setuptools
+% pip install `.[tests,dev]
+```
+
+The extra options `tests` and `dev` are not strictly necessary for publishing,
+but are handy to have installed while working on the schema definitions.
+Once installed publishing could be as simple as running:
+
+```console
+% publish
+```
+
+but it likely isn't.
+
+See, the `publish` tool expects a number of environment variables to be set.
+These are:
+
+```console
+DATAPUNT_ENVIRONMENT=...
```

Comment:
Aangeven wat we hier kunnen invullen o.i.d.?
DATAPUNT_ENVIRONMENT=[acceptation|production]

Reply:
Will do.Thanks!